js/web/lib/wasm/jsep/webgpu/ops/3rd-party/conv_backprop_mm_webgpu.ts

```diff
@@ -209,42 +224,23 @@ export const createConv2DTransposeMatMulProgramInfo =
 
       programUniforms.push(...createTensorShapeVariables(outputShape));
 
+      const uniforms: UniformsArrayType = [
+        {name: 'dimAOuter', type: 'i32'}, {name: 'dimBOuter', type: 'i32'}, {name: 'dimInner', type: 'i32'},
+        {name: 'strides', type: 'i32', length: 2}, {name: 'dilations', type: 'i32', length: 2},
+        {name: 'filterDims', type: 'i32', length: filterDims.length}, {name: 'pads', type: 'i32', length: pads.length}
+      ];
+
       return {
         name: 'Conv2DTransposeMatMul',
-        shaderCache: {hint: attributes.cacheKey},
+        shaderCache: {hint: `${attributes.format}`, inputDependencies},
```

Comment:
missing `workGroupSize`/`elementsPerThread` in hint?

Reply:
workGroupSize is constaant, add elementsPerThread in shader key.